tests/test_command_handler.py

```diff
@@ -101,13 +101,15 @@ async def test_get_telemetrey(self):
         self.assertEqual(self.response["response"], ResponseCode.OK)
         self.assertTrue(self.command_handler._started)
 
+        # Give some time to the mock sensor to produce data
```

Comment:
Thanks. But...is there some way to detect this directly, e.g. by awaiting something in the mock sensor. Or if you also need for it to be read by "the thing that reads a sensor" then you could await something that that "thing". It's much more robust that just sleeping.

Reply:
The real sensors also don't report when telemetry is available so this is "natural" behavior. I agree that sleeping isn't very robust so I am open to suggestions for how to get rid of the sleep.

Reply:
If you build a mock object it can have attributes for things like this, such as a "start_task" that is done when the class is connected and producing telemetry. I think that sort of thing is very appropriate for unit tests. It's not realistic, but the unrealistic part is tangential to what you are testing -- it makes the test more efficient and robust.